db/schema.rb

```diff
@@ -398,6 +399,7 @@
   add_index "users", ["display_name"], name: "users_display_name_trgm_index", using: :gist, opclass: {"display_name"=>:gist_trgm_ops}
```

Comment:
@edpaget what's the use case for fuzzy matching on the display_name now? All @ mentions just using login field now.